### ISSUE TITLE
[Bug] Shortcuts not work in cmder, and alias names aren't coloured correctly.

### ISSUE THREAD

**Issue by frankpfan** [cmderdev/cmder#2920]
### Version Information

```markdown
Cmder version: newest
Platform: Windows10
```

### Cmder Edition

Cmder Full (with Git)

### Description of the issue

*(Excuse my terrible English because I'm a Chinese student :)*

I'm having the problem on my Windows10 cmder full with ConEmu.

I'm used to pressing `Ctrl + Alt + U` to `cd ..`. It has always worked fine, but not today.

I have another two problems meanwhile.

1. Cmder doesn't show the current git branch in the prompt. When I'm in a git repo, it fails to recognise and doesn't show (main -> origin). However, git works fine.
2. Aliases aren't coloured in yellow. I use monokai color scheme, which shows alias names in a different colour. But now it's wrong. However, all alias commands are right.

So all of these seem to be bugs in cmder itslef (or maybe ConEmu?). They came to me at the same time. But there were no problems when I'd been using cmder just a short time ago. I suppose they're related?

**Additional Information**: When I open cmder, a line of message appears and immediately vanishes, showing that it can't find clink_x64.exe. I only have clink_x86.exe and clink_arm64.exe in the right place. Would it be one cause?

### How to reproduce

Sorry, I didn't do anything with cmder's files...

### Checklist

- [X] I have read the documentation.
- [X] I have searched for similar issues and found none that describe my issue.
- [X] I have reproduced the issue on the latest version of Cmder.
- [X] I am certain my issues are not related to ConEmu, Clink, or other third-party tools that Cmder uses.

**Comment by frankpfan** [comment]
#2832 is a similar issue, but that's more likely to be a problem with ConEmu, not cmder. I wonder if they have similar ways to solve, though.

**Comment by chrisant996** [comment]
Yes, the cause of all of the problems you described is the fact that you are running on 64-bit Windows, but you don’t have clink_x64.exe.  Clink is what provides Ctrl-Alt-U, the git branch in the prompt, and colors aliases.

Check your anti-virus software logs:  I would bet that your anti-virus software got concerned about clink_x64.exe and deleted it.  Clink can’t do anything about that.  If that's what happened, you would need to contact your anti-virus software's support team or site, and report that the anti-virus software detected a false positive.

You can get the file back by downloading the latest Clink release .zip file and extracting the files.  Also make sure the clink_dll_x64.dll file isn't gone, nor any other files normally included in Clink.  It's safe to overwrite any of the files that already exist; none of them get modified in any way later.

**Comment by chrisant996** [comment]
> #2832 is a similar issue, but that's more likely to be a problem with ConEmu, not cmder. I wonder if they have similar ways to solve, though.

No, 2832 is completely unrelated.  I saw this was originally added as a reply in 2832, and I was going to say thank you for opening a new issue instead, since it's completely unrelated.

**Comment by frankpfan** [comment]
Yeah i'll try it out. Thank u @chrisant996!

**Comment by chrisant996** [comment]
Also, see [clink issue 369](https://github.com/chrisant996/clink/issues/369) which is a pinned issue about anti-virus interference.

**Comment by frankpfan** [comment]
sorry about #2832 , my eyes need some help...
